deis/azuredeploy.json

```diff
@@ -0,0 +1,211 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "String",
+            "defaultValue": "West US",
```

Comment:
Remove default value and have allowed values like here https://github.com/Azure/azure-quickstart-templates/blob/master/101-create-availability-set/azuredeploy.json